package.json

```diff
@@ -65,11 +65,11 @@
     "nock": "^11.7.0",
     "prettier": "^1.16.4",
     "proxyquire": "^2.1.3",
-    "semantic-release": "^19.0.0",
+    "semantic-release": "^20.1.0",
```

Comment:
I would not change this and avoid a breaking change.

Reply:
You mean that we should avoid upgrading the semantic-release then?

Reply:
This change isn't related to the consumers of semantic-release-slack-bot, as it's just a `devDependency`. What concern do you see with this version bump @tripodsan ?

Reply:
ah right. maybe it's not a problem.